packages/openapi-v3/src/json-to-schema.ts

```diff
@@ -94,6 +94,13 @@ export function jsonToSchemaObject(json: JsonSchema): SchemaObject | SchemaRef {
         );
         break;
       }
+      case 'examples': {
+        if (Array.isArray(json.examples)) {
+          result.examples = json.examples;
+          result.example = json.examples[0];
```

Comment:
I am not familiar with JSON Schema. Is it possible to specify a single example in JSON Schema too? Should we check if `result.example` has not been set by other means so that we don't overwrite it with the first item of examples array?

Reply:
`example` is not allowed in `JsonSchema`, so i think this can not happen. If this would be allowed we would not need this lines / workaround at all.

Reply:
Makes sense, thank you for the explanation 👍